### ISSUE TITLE
Dividing a long video into multiple 10 sec clips when using Gallery Override.

### ISSUE THREAD

**Issue by gigacastelo** [rhunk/SnapEnhance#96]
### Description

I think it would be nice if there was an option to send a long video from gallery (Using Gallery Override) but making sure that it sends as multiple 10 sec vids. As if it was a normal snap video. It would seem more real to whoever receives the snap, since Snapchat does not send long videos as a one single snap.
### Agreement

- [X] There is no issue already suggesting my feature.

**Comment by RevealedSoulEven** [comment]
+1

**Comment by RevealedSoulEven** [comment]
The thing is that no videos should be merged until "Disable snap splitting" is turned on. Either it is snap or gallery.

**Comment by gigacastelo** [comment]
Yes I agree. It doesn't make sense that it has a "disable Snap splitting"
option if it's already disabled by default.

**Comment by rhunk** [comment]
Duplicate of #112

**Comment by rhunk** [comment]
Fixed video splitting https://github.com/rhunk/SnapEnhance/commit/3cd628f7cfa326e02b49f3804a5dbd07bc74ab96
